model_compression_toolkit/qunatizers_infrastructure/pytorch/pytorch_node_quantization_dispatcher.py

```diff
@@ -0,0 +1,42 @@
+from typing import Dict, List
+
+from model_compression_toolkit.core.common import Logger
+from model_compression_toolkit.core.common.constants import FOUND_TF
+from model_compression_toolkit.qunatizers_infrastructure.common.base_quantizer import BaseQuantizer
+from model_compression_toolkit.qunatizers_infrastructure.common.node_quantization_dispatcher import \
+    NodeQuantizationDispatcher
+from model_compression_toolkit.core.common.constants import FOUND_TORCH
+
+if FOUND_TORCH:
+    ACTIVATION_QUANTIZERS = "activation_quantizers"
+    WEIGHT_QUANTIZERS = "weight_quantizer"
+
+
+    class PytorchNodeQuantizationDispatcher(NodeQuantizationDispatcher):
```

Comment:
Why this class doesn't implement get_config and from_config like the parallel one in Keras?

Reply:
it's only needed for Keras